config/typescript.js

```diff
@@ -19,4 +19,8 @@ module.exports = {
     },
   },
 
+  rules: {
+    // analysis/correctness
```

Comment:
the comment here should explain why it's disabled, i think

Reply:
Good point, I added this as thee "category" header same as https://github.com/benmosher/eslint-plugin-import/blob/master/config/recommended.js#L9 but I can also do that!

Reply:
@ljharb, done, what do you think?